tests/scsi/010

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2025 Google LLC
+
+. tests/scsi/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test F2FS data lifetime support"
```

Comment:
This description and the commit message title both confused me (why not fstests?). So I suggest "test write hint  integration with F2FS data lifetime".